src/backer/crawling.py

```diff
@@ -14,11 +14,12 @@ class to support subcribe tip from ogmios and cardano node
 
 class Crawler(doing.DoDoer):
 
-    def __init__(self, on_tip=False, backer=None, **kwa):
+    def __init__(self, backer, ledger, on_tip=False, **kwa):
```

Comment:
Actually we should never pass `on_tip` to the constructor right? It should always start crawling and verify itself if its on tip or not, and have a default in the constructor as false. (and not a constructor parameter - so just `self.on_tip = False`)